client/dashboard/profile-wizard/steps/plugins.js

```diff
@@ -2,50 +2,199 @@
 /**
  * External dependencies
  */
-import { __ } from '@wordpress/i18n';
+import { __, sprintf } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
 import { Component, Fragment } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
+import { forEach } from 'lodash';
+import { compose } from '@wordpress/compose';
+import { withDispatch } from '@wordpress/data';
 
 /**
  * Internal depdencies
  */
-import { H, Stepper } from '@woocommerce/components';
+import { H, Stepper, Card } from '@woocommerce/components';
 import ProfileWizardHeader from '../header';
 
-export default class Start extends Component {
+const plugins = [ 'jetpack', 'woocommerce-services' ];
+
+class Plugins extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.state = {
+			step: 'install',
+			isPending: true,
+			isError: false,
+			pluginsInstalled: 0,
+			pluginsActivated: 0,
+		};
+
+		this.activatePlugins = this.activatePlugins.bind( this );
+	}
+
+	componentDidMount() {
+		this.installPlugins();
+	}
+
+	componentDidUpdate( prevProps, prevState ) {
+		if (
+			this.state.pluginsInstalled !== prevState.pluginsInstalled &&
+			this.state.pluginsInstalled === plugins.length
+		) {
+			/* eslint-disable react/no-did-update-set-state */
+			this.setState( {
+				step: 'activate',
+				isPending: false,
+			} );
+			/* eslint-enable react/no-did-update-set-state */
+		}
+
+		if (
+			this.state.pluginsActivated !== prevState.pluginsActivated &&
+			this.state.pluginsActivated === plugins.length
+		) {
+			window.location = wcSettings.jetpackConnectUrl;
+		}
+	}
+
+	installPlugins() {
+		forEach( plugins, async plugin => {
+			const response = await this.installPlugin( plugin );
+			if ( 'success' === response.status ) {
+				this.setState( state => ( {
+					pluginsInstalled: state.pluginsInstalled + 1,
+				} ) );
+			}
+		} );
+	}
+
+	activatePlugins() {
+		this.setState( {
+			isPending: true,
+		} );
+
+		forEach( plugins, async plugin => {
+			const response = await this.activatePlugin( plugin );
+			if ( 'success' === response.status ) {
+				this.setState( state => ( {
+					pluginsActivated: state.pluginsActivated + 1,
+				} ) );
+			}
+		} );
+	}
+
+	async installPlugin( plugin ) {
```

Comment:
Completely optional, but I feel like we could dry up this and `activatePlugin()` since there are only a couple minor differences.  Will defer to your judgement as to whether or not it's worth it.

Reply:
Done! This has been cleaned up.